ceremonies/assignment/src/math.rs

```diff
@@ -110,19 +127,19 @@ pub fn find_random_coprime_below<H: Hash>(
 		.unwrap_or(1)
 }
 
-pub fn mod_inv(a: i64, module: i64) -> i64 {
+pub fn mod_inv(a: i64, module: i64) -> Option<i64> {
```

Comment:
To stay with the naming conventions, this should be renamed to `checked_mod_inv`

Reply:
I am confused, you resolved the conversation without doing anything? Did you forget to push? :)